trade-finance-manager-api/src/fileshare-test.js

```diff
@@ -3,22 +3,24 @@ const isPortReachable = require('is-port-reachable');
 const urlList = ['rrstagingmedia.file.core.windows.net', 'google.com'];
 
 const fetchTest = () =>
-  urlList.map(async (url) => {
-    const port443 = await isPortReachable(443, { host: url });
-    const port80 = await isPortReachable(80, { host: url });
-    const port445 = await isPortReachable(445, { host: url });
-    const port10101 = await isPortReachable(10101, { host: url });
-    return {
-      url,
-      port80,
-      port443,
-      port445,
-      port10101,
-    };
-  });
+  Promise.all(
+    urlList.map(async (url) => {
+      const port443 = await isPortReachable(443, { host: url });
+      const port80 = await isPortReachable(80, { host: url });
+      const port445 = await isPortReachable(445, { host: url });
+      const port10101 = await isPortReachable(10101, { host: url });
+      return {
+        url,
+        port80,
+        port443,
+        port445,
+        port10101,
+      };
+    }),
+  );
 
 const shareTest = async () => {
-  const ports = fetchTest();
+  const ports = await fetchTest();
```

Comment:
`fetchTest` currently isn't marked with `async`, so `await` might be highlighted as an error in the future. This should be an easy fix though as we just need to mark that function as `async`

Reply:
marked